pkg/investigations/investigation/investigation.go

```diff
@@ -20,6 +21,10 @@ type InvestigationStep struct {
 }
 
 type InvestigationResult struct {
+	// NEW: Actions to execute via reporter (modern approach)
+	Actions []types.Action
+
+	// EXISTING: Legacy fields (deprecated, maintained for backwards compatibility)
 	LimitedSupportSet  InvestigationStep
 	ServiceLogPrepared InvestigationStep
```

Comment:
thought: do we keep something equivalent to servicelog prepared? 

I think we could formalize the "informing" state of an investigation pretty easily after this change. 
If the investigation is marked as informing, the actions are not executed but only posted to pd?  
And then we wont need the servicelog perpared anymore